app/views/layouts/_styles.html.erb

```diff
@@ -3,12 +3,14 @@
     <style>
       <% Rails.application.config.assets.compile = true %>
       <%= Rails.application.assets["crayons.css"].to_s.html_safe %>
+      <%= Rails.application.assets["views.css"].to_s.html_safe %>
```

Comment:
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨

Now this is pretty important....

I took some views-oriented styling out of Crayons and put it in a separate file. This will be pretty important when we actually get to making Crayons a standalone gem. The idea is to gem be very generic and provide **only** basic styling and components. Styling for particular views should not be part of Crayons.